CHANGELOG.md

```diff
@@ -1,3 +1,931 @@
+# 83.0.0
+
+This major release graduates several Theming extensions from Beta and includes a breaking change for
+AppBar's imports. All Themer APIs have been annotated as "to be deprecated" in lieu of Theming
+extensions.
```

Comment:
How do you feel about listing all the components that are graduating their theming extension?

Reply:
Done.